javascripts/discourse/components/tag-game-card-info-item.gjs

```diff
@@ -0,0 +1,31 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { inject as service } from "@ember/service";
+import dIcon from "discourse-common/helpers/d-icon";
+
+export default class TagGameCardInfoItem extends Component {
+  @service router;
+  @tracked icon;
+  @tracked info;
+  @tracked text;
+
+  constructor() {
+    super(...arguments);
+    this.icon = this.args.icon;
+    this.info = this.args.info;
+    this.text = this.args.text;
```

Comment:
nitpick: you don't need these, you can use `@icon`, `@info` and `@text` directly in the template. Indeed, you're not using the `router` either, so this file could be just the template